src/plugins/credential_management/public/components/credential_table/credentials_table.tsx

```diff
@@ -162,6 +153,28 @@ export const CredentialsTable = ({ canSave, history }: Props) => {
     );
   };
 
+  const renderToolsRight = () => {
```

Comment:
Let's add more comments and organize the code

Reply:
Sure. Have fixed it.